sentry/const.py

```diff
@@ -34,10 +35,10 @@ def split_multiple(string, delimiter=',', strip_chars=None):
 
 DEFAULT_TRANSPORT = 'threaded'
 TRANSPORT_CLASS_MAP = {
-    'requests_synchronous': raven.transport.RequestsHTTPTransport,
-    'requests_threaded': raven.transport.ThreadedRequestsHTTPTransport,
-    'synchronous': raven.transport.HTTPTransport,
-    'threaded': raven.transport.ThreadedHTTPTransport,
+    'requests_synchronous': getattr(raven, 'transport.RequestsHTTPTransport', False),
```

Comment:
or add raven_import = True in try and False in exception and then add a conditional dict